.github/workflows/workflow.yml

```diff
@@ -159,6 +161,7 @@ jobs:
           stack-name: dev
           upsert: true
           work-dir: .github/test-stacks/nodejs
+          configMap: "{name: {value: hostname, secret: false}}"
```

Comment:
I am wondering if GitHub Actions would let us use explicit subtree here instead of  encoding with "", such as:

```
configMap:
  name:
    value: hostname
    secret: false
```

Reply:
@t0yv0 having a hard time confirming in official documentation, but according to [this post](https://github.community/t/can-action-inputs-be-arrays/16457) on the community forum only strings are supported for action inputs.

Reply:
You're right I think it's here https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs